### DISCUSSION TITLE
correspondence history

### DISCUSSION THREAD

**Discussion by ArtemGurdjiev** [community/community#123933] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is it possible to store all correspondence history in the LinkedIn Sales Navigator integration or can we only see the last message as implemented in Hubspot?

**Comment by ebndev** [comment]
Hi @ArtemGurdjiev, thanks for being a part of the GitHub Community!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place. Please feel free to provide more information about your post if you **did** intend to post in Discussion's public forum.

Best of luck!
